src/gui/mzroll/database.cpp

```diff
@@ -874,23 +874,15 @@ int Database::loadCompoundCSVFile(string filename)
         if ( mz > 0 || ! formula.empty() || precursormz > 0 || mass > 0.0f) {
             Compound* compound = new Compound(id,name,formula,charge);
 
-            compound->expectedRt = rt;
-
-            if (mass != 0.0f)
-                compound->neutralMass = mass;
-
-            if (mz == 0 && !formula.empty()) {
-                mz = MassCalculator::computeMass(formula,charge);
-            } else if (mass != 0.0f) {
-                mz = MassCalculator::adjustMass(mass, charge);
-            }
```

Comment:
No, this was not intentional. May be because of the initial rebasing, these checks had been removed and same was with neutral mass. I had added yesterday where ever i got them. But still which were left I have added now after your review